### PR TITLE
Form generator

### PR DESCRIPTION
**Pull request by ekazankova**
https://app.clickup.com/t/24336023/PRD-1208

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -164,6 +164,27 @@ Creates an API with related entities in API library. It also updates redux store
...
+   (e.g: `ProfileSettings` or `useProfileSettings`)
+
+#### Example
+
+```sh
+npx nx g form --name=profile-settings --placeOfUse=ProfileSettings
+```
+or
+```sh
+npx nx g expo-app profile-settings ProfileSettings
```
Comment (kerne1hub) [inline README.md:185]:
```suggestion
npx nx g form profile-settings ProfileSettings
```

File: libs/mobile/shared/features/settings/src/lib/forms/settings.ts
Comment (kerne1hub) [inline libs/mobile/shared/features/settings/src/lib/forms/settings.ts:1]: Can we remove <b>libs/mobile</b> directory?

File: plugin/README.md [hunk 2]
```diff
@@ -164,6 +164,27 @@ Creates an API with related entities in API library. It also updates redux store
...
+   (e.g: `ProfileSettings` or `useProfileSettings`)
+
+#### Example
+
+```sh
+npx nx g form --name=profile-settings --placeOfUse=ProfileSettings
+```
+or
+```sh
+npx nx g expo-app profile-settings ProfileSettings
```
Comment (kerne1hub) [inline plugin/README.md:185]:
```suggestion
npx nx g form profile-settings ProfileSettings
```

File: plugin/src/generators/next-app/generator.ts [hunk 3]
```diff
@@ -47,6 +47,12 @@ export async function nextAppGenerator(
     await runApiClientGenerator(tree, options);
   }
 
+  const shouldGenerateFormUtilsLib = await askQuestion('Do you want to create a lib with the form  utils? (y/n): ') === 'y';
```
Comment (kerne1hub) [inline plugin/src/generators/next-app/generator.ts:50]:
```suggestion
  const shouldGenerateFormUtilsLib = await askQuestion('Do you want to create a lib with the form utils? (y/n): ') === 'y';
```

File: plugin/src/shared/generators/form-utils/generator.ts [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+import { formatFiles, generateFiles, Tree } from '@nx/devkit';
+
+export async function runFormUtilsGenerator(
+  tree: Tree,
+  options: { name: string; directory: string }
+) {
+  const libRoot = `libs/${options.directory}`;
+
+  // Generate shared libs
+  execSync(`npx nx g react-lib ${options.directory}/shared/utils/form`, { stdio: 'inherit' });
```
Comment (kerne1hub) [inline plugin/src/shared/generators/form-utils/generator.ts:12]: could you please update the command args? The option `directory` was removed in https://github.com/RonasIT/nx-generators/pull/37. You can use `app=shared` option instead.

**Comment by ekazankova** [comment]
I also changed the order of generated libs in `expo-app` and `next-app` and documentation for `react-lib`

**Comment by kerne1hub** [comment]
@ekazankova looks good to me

**Pull request merged**